src/components/team/member-table.tsx

```diff
@@ -1,59 +1,98 @@
+import * as React from 'react'
 import {format} from 'date-fns'
+import {useMachine} from '@xstate/react'
+import confirmationDialogMachine from 'machines/confirmation-dialog-machine'
+import RemoveMemberConfirmDialog from 'components/team/remove-member-confirm-dialog'
+
+const MemberTable = ({
+  accountId,
+  members,
+  setMembers,
+}: {
+  accountId: number
+  members: any[]
+  setMembers: any
+}) => {
+  const initialContext = {accountId, setMembers}
+  const [current, send] = useMachine(confirmationDialogMachine, {
+    context: initialContext,
+  })
```

Comment:
Initialize the XState machine that powers the Confirm dialog. The initial context includes the `accountId` (needed for constructing the DELETE request URL) and `setMembers` (the setter for updating the list of members displayed on the page).